docs/platform/webcontainers/turbo-package-manager.md

```diff
@@ -10,6 +10,12 @@ head:
 
 # {{ $frontmatter.title }}
 
+:::warning
+Starting on January 1, 2024, turbo will no longer be supported and will be removed. We now offer built-in support for npm, pnpm, and yarn.
+
+Adding support for all the features from other package managers into turbo was a lot of work. That's why we decided to focus on supporting the native package managers out of the box instead of building our own.
```

Comment:
```suggestion
Adding support for all the features from other package managers into turbo required a lot of continuous work. Because of that, we now support native package managers instead of building our own.
```